tools/localization_analysis/scripts/auto_run.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+import argparse
+import subprocess
+import sys
+
+import rosbag
+
+parser = argparse.ArgumentParser(
+    description="This file is intended to run the ground truth scripts on mutiple bags in the same directory with one map"
+)
+
+# define arguments
+parser.add_argument(
+    "--bag_path", "-bp", type=str, help="path of the directory containing your bags"
+)
+
+parser.add_argument(
+    "--map_path", "-m", type=str, help="path of the directory containing your map"
+)
+
+parser.add_argument(
+    "--bags", "-b", nargs="+", type=str, help="List of bag names to process"
```

Comment:
I would make bag_path -b, and turn --bags into a positional argument (meaning you don't need a flag at all, just put the files on the command line standalone)